cuqi/utilities/_utilities.py

```diff
@@ -64,7 +64,7 @@ def get_indirect_variables(dist):
     attributes = []
     for attribute in dist.get_mutable_variables():
         value = getattr(dist, attribute)
-        if callable(value):
+        if callable(value) and not isinstance(value, spslinalg.LinearOperator):
```

Comment:
First "callable" object that we want to interpret as a fix variable. I suggest we add a method to check if "value" is one of the allowed cases that are callable but still act as a well-defined ndarray

Reply:
Added a method `check_if_conditional_from_attr` in `cuqi/utilities/_utilities.py` for your suggested job

Reply:
LGTM